README.md

```diff
@@ -1,23 +1,25 @@
-# InfluxData Helm Charts
+# InfluxData Helm charts
 
 [![License](https://img.shields.io/badge/license-MIT-green.svg)](./LICENSE)
 [![](https://github.com/influxdata/helm-charts/workflows/helm-charts%2Frelease/badge.svg?branch=master)](https://github.com/influxdata/helm-charts/actions)
 
-
-This functionality is in beta and is subject to change. The code is provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+InfluxData Helm charts are in beta and subject to change. The code is provided as-is with no warranties. Beta features are not subject to the support service level agreement (SLA) of general availability (GA) features.
```

Comment:
is "support service level agreement" an actual doc?... or are we saying something about supporting SLA?

Reply:
No idea, it was added by an external contributor and I assume copied from a similiar repo.

https://github.com/influxdata/helm-charts/commit/ddee0bb7600051e558a4feefc4767ead8f334205

Happy to remove or adjust, whatever you think is best

Reply:
thanks @rawkode - updated note to refer to our Software Subscription agreement and included link.